impl/src/main/java/dao/AnswerDao.java

```diff
@@ -48,4 +46,12 @@
 
     @Update(value = "UPDATE answer SET votes=votes-1 WHERE slack_id = :threadId")
     Observable<Void> downVoteAnswer(String threadId);
+
+    @Query("SELECT a.id, a.user_id, a.answer, a.created_at, a.accepted, a.title, a.votes, a.slack_id, a.question_id, u.name AS created_by, " +
```

Comment:
it strikes me now, when reading this, that we should indent the sql query. Its hard to read. Can we maybe place the select in a variable for reuse for the other select queries as well?

Reply:
I will try to improve formatting.